src/goom_config.h

```diff
@@ -2,10 +2,10 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
+#define GOOM_TEXTURE_WIDTH  455
```

Comment:
What do you thing to use direct the resolutions 1280x720 and 640x360, sizes designed to match bit sizes.

With 512x455 you are right is really bad (455/8=56,875; 512/8=64)

With 1280x720 is it always divisible (720/8=90; 1280/8=160)
and 640x360 is always divisible (360/8=45; 640/8=80)